nmap/plan.sh

```diff
@@ -9,14 +9,55 @@ pkg_source=https://nmap.org/dist/${pkg_name}-${pkg_version}.tar.bz2
 pkg_shasum=a8796ecc4fa6c38aad6139d9515dc8113023a82e9d787e5a5fb5fa1b05516f21
 pkg_deps=(
   core/glibc
-  core/zlib
-  core/nghttp2
   core/gcc-libs
   core/openssl
+  core/pcre
+  core/zlib
 )
 pkg_build_deps=(
+  core/bzip2
   core/coreutils
+  core/diffutils
+  core/file
   core/gcc
+  core/inetutils
+  core/lua
   core/make
+  core/openssh
   core/pkg-config
+  core/readline
+  core/which
 )
+
+do_prepare() {
+  export CFLAGS="${CFLAGS} -O -Wcpp"
```

Comment:
What do you think about going with `-O2` if we are turning on optimizations.

Reply:
I don't know tbh. The build kept sprouting warnings about using `-O` and optionally `-Wcpp` that's why I added them ;) If you think `-O2` is better, then so be it :)